### PR TITLE
LIVE-2693 create index in real time instead of loading from app.json for btc storage

### PR DESCRIPTION
**Pull request by hzheng-ledger**
### 📝 Description

In wallet btc, the unspent utxo and index was loaded from app.json before. Such indexes were broken because of unknown reason sometimes. These broken indexes can cause incorrect bitcoin balance or other issues(txs[index] undefined).
I make this PR to generate such indexes in real time. So that it may fix incorrect utxo and incorrect balance bugs.

### ❓ Context

- **Impacted projects**:  None
- **Linked resource(s)**:  
https://ledgerhq.atlassian.net/browse/LIVE-2693 
https://ledgerhq.atlassian.net/browse/LIVE-2495 
https://ledgerhq.atlassian.net/browse/TECSUPPORT-1220

### ✅ Checklist

- [ ] **Test coverage** 
- [X] **Atomic delivery** 
- [X] **No breaking changes** 

### 📸 Demo

No demo for this issue as the change is supposed to be transparent to end users.

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: .changeset/lucky-bugs-shake.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@ledgerhq/live-common": major
```
Comment (haammar-ledger) [inline .changeset/lucky-bugs-shake.md:2]: This should be `minor`. Thanks.

File: libs/ledger-live-common/src/families/bitcoin/wallet-btc/storage/index.ts [hunk 2]
```diff
@@ -78,13 +78,10 @@ class BitcoinLikeStorage implements IStorage {
...
       }
-      if (
-        typeof this.accountIndex[`${tx.account}-${tx.index}`] === "undefined"
-      ) {
-        this.accountIndex[`${tx.account}-${tx.index}`] = [];
-      }
       const idx = this.txs.push(tx) - 1;
       this.primaryIndex[index] = idx;
+      this.accountIndex[`${tx.account}-${tx.index}`] =
+        this.accountIndex[`${tx.account}-${tx.index}`] || [];
```
Comment (haammar-ledger) [inline libs/ledger-live-common/src/families/bitcoin/wallet-btc/storage/index.ts:84]: `??` is safer than `||` in this kind of case, but probably not very important here.

**Review by haammar-ledger — CHANGES_REQUESTED** [review]
We should try to add a unit test that calls the clear account function.

File: libs/ledger-live-common/src/families/bitcoin/wallet-btc/storage/index.ts (hunk 2 above)
Comment (hzheng-ledger) [reply libs/ledger-live-common/src/families/bitcoin/wallet-btc/storage/index.ts:84]: It is the same here. but as in the file, we use all "||" to handle this kind of logic. I usee "||" here too

**Comment by haammar-ledger** [comment]
@hzheng-ledger thanks for the changeset, what do you think about this one?

> Maybe we should try to add a unit test that calls the `clearAccount` function.

**Review by gre — APPROVED** [review]

**Pull request merged**